.env.example

```diff
@@ -9,3 +9,4 @@ GA4_CONTAINER_ID="GTM-KD86CMZ"
 ANALYTICS_COOKIE_DOMAIN="localhost"
 LANGUAGE_TOGGLE_DISABLED=false
 ANALYTICS_DATA_SENSITIVE=false
+LOG_LEVEL=
```

Comment:
I left it blank because there is a fault in the config file it's not something that is required, it would only be helpful if someone wants to add their own value